geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -18,7 +18,14 @@
     <!-- document details -->
 
     {# title #}
-    <h1>{{ document.title }}</h1>
+    <h1>
+        {% if document.doctype %}
+            <span class="doctype">{{ document.doctype }}</span>
+        {% else %}
+            <span class="doctype">Unknown type</span>
+        {% endif %}
```

Comment:
This can be simpler too, actually, although I'm not sure how you integrate it with the translation (maybe need to get the translation text first).
```suggestion
         <span class="doctype">{{ document.doctype|default:"Unknown type" }}</span>
```